osu.Game/Screens/Play/HUD/SkinnableInfo.cs

```diff
@@ -32,6 +32,9 @@ public class SkinnableInfo : IJsonSerializable
 
         public Anchor Origin { get; set; }
 
+        /// <inheritdoc cref="ISkinnableDrawable.UsesFixedAnchor"/>
```

Comment:
We generally don't use `inheritdoc`. It's automatically handled in IDEs these days without its presence.

Reply:
Okay so this isn't actually inheriting from a direct interface. Just add local documentation here then, probably referencing how it is consumed/sourced from the related property on the interface.

Reply:
This documentation should mention that this is only used in the skin layout editor context.

I'd almost be inclined to invert the naming to `UseClosestAnchor` too..

Reply:
> I'd almost be inclined to invert the naming to `UseClosestAnchor` too..

This commit (on a [separate branch](https://github.com/ribbanya/osu/tree/skin-editor-closest-anchor-invert)) will do that: https://github.com/ppy/osu/commit/7dc8f9aafa165d36c7604dd7b70223ea5970bb9d

The serialization does work, I just think the default-true semantics are a little wonky when implementing the interface. Up to you.

Reply:
That branch presents an issue: the default implementations should not be specifying `true` for this property. It feels completely weird.

Reply:
Yeah, exactly. I'm not aware of a better way to invert the property. Will implement if anyone has any suggestions. Otherwise, I guess it will have to be `UsesFixedAnchor`.

The fundamental problem is that when an `ISkinnableDrawable` which has never been serialized is encountered by the editor, the only information it has is the CLR default value for the property. Which means that if we want "closest anchor" behaviour to be the default, and `UsesClosestAnchor` is the name of the property, that property must be `true` when a new element is instantiated.

Reply:
I think it's okay with the behaviour inverted. I'm hoping we can find a better way of defining this skinnable interface in a future, but it should be fine for now to get this functionality actually in the game.